src/scripts/optimistic-tock.js

```diff
@@ -2,88 +2,102 @@ const holidays = require("@18f/us-federal-holidays");
 const moment = require("moment-timezone");
 const scheduler = require("node-schedule");
 const {
+  optOut,
   slack: { sendDirectMessage },
   tock: { get18FTockTruants, get18FTockSlackUsers },
 } = require("../utils");
 
 const TOCK_API_URL = process.env.TOCK_API;
 const TOCK_TOKEN = process.env.TOCK_TOKEN;
 
-const reminder = (tz) => async () => {
-  const message = {
-    username: "Happy Tock",
-    icon_emoji: "happytock",
-    text: "Don't forget to <https://tock.18f.gov|Tock your time>!",
-  };
-
-  // Get all the folks who have not submitted their current Tock.
-  const truants = await get18FTockTruants(moment.tz(tz), 0);
+module.exports = async (app) => {
```

Comment:
I recommend reviewing this with whitespace changes disabled. I moved all of the function declarations into the exported function instead of being module-level functions. The reason is that two of them needed access to the `output` variable, but I didn't want that to be module-level to simplify testing.

If you turn off whitespace in the GitHub review thingy, I think it becomes clearer that not a lot really changed here.